pkg/monitoring/monitoring.go

```diff
@@ -57,6 +57,9 @@ const (
 	CattlePrometheusRuleLabelKey             = "source"
 	CattleAlertingPrometheusRuleLabelValue   = "rancher-alert"
 	CattleMonitoringPrometheusRuleLabelValue = "rancher-monitoring"
+
+	rancherMonitoringTemplateName = "system-library-rancher-monitoring"
+	catalogTemplatesNamespace     = "cattle-global-data"
```

Comment:
you can use `pkg/namespace/namespace_utils.go:       GlobalNamespace = "cattle-global-data"`